### PR TITLE
Feat/more tests 4 update

### PR DESCRIPTION
**Pull request by tonsV2**

### CODE REVIEW

File: pkg/database/handler.go [hunk 1]
```diff
@@ -411,7 +410,7 @@ func (h Handler) Unlock(c *gin.Context) {
 	}
 
 	if d.Lock == nil {
-		_ = c.AbortWithError(http.StatusNoContent, errors.New("database not locked"))
+		c.String(http.StatusNoContent, "database not locked")
```
Comment (teleivo) [inline pkg/database/handler.go:413]: that change does not relate to the test you added, right?

File: pkg/database/handler_test.go [hunk 2]
```diff
@@ -21,6 +21,38 @@ import (
...
+	repository.
+		On("Update", database).
+		Return(nil)
+	service := NewService(config.Config{}, nil, nil, repository)
+	handler := New(nil, service, nil)
+
+	w := httptest.NewRecorder()
+	c, _ := gin.CreateTestContext(w)
+	c.AddParam("id", "1")
+	user := &models.User{
```
Comment (teleivo) [inline pkg/database/handler_test.go:39]: Are you going to use the helper we talked about 😋 

```go
func newTestContext(w *httptest.ResponseRecorder, userID uint64, group string) *gin.Context {
    c, _ := gin.CreateTestContext(w)
    user := &models.User{
        ID: userID,
        Groups: []*models.Group{
            {Name: group},
        },
    }
    c.Set("user", user)
    return c
}
```

File: pkg/database/handler.go (hunk 1 above)
Comment (tonsV2) [reply pkg/database/handler.go:413]: No, it was added by a mistake and I didn't want to open a separate PR

Comment (teleivo) [reply pkg/database/handler.go:413]: @tonsV2 got it. Please commit things like that separately though. IMHO you can push that straight to master. A PR should contain a cohesive piece of work. This makes it easier to review.

File: pkg/database/handler_test.go [hunk 3]
```diff
@@ -21,6 +21,31 @@ import (
...
+		On("FindById", uint(1)).
+		Return(database, nil)
+	repository.
+		On("Update", database).
+		Return(nil)
+	service := NewService(config.Config{}, nil, nil, repository)
+	handler := New(nil, service, nil)
+
+	w := httptest.NewRecorder()
+	c := newContext(w, "group-name")
```
Comment (teleivo) [inline pkg/database/handler_test.go:37]: will you refactor the other tests to use `newContext` later? 😄

**Review by teleivo — APPROVED** [review]

File: pkg/database/handler.go (hunk 1 above)
Comment (tonsV2) [reply pkg/database/handler.go:413]: :+1:

**Pull request merged**